scripts/world_bank/wdi/output/WorldBank_StatisticalVariables.mcf

```diff
@@ -634,3 +634,163 @@ statType: dcs:measuredValue
 measuredProperty: dcs:amount
 transferType: dcs:OutwardRemittance
 
+
+Node: dcid:WorldBank/SH_DYN_MORT
+name: "Mortality rate, under-5 (per 1,000 live births)"
+description: "Under-five mortality rate is the probability per 1,000 that a newborn baby will die before reaching age five, if subject to age-specific mortality rates of the specified year. Estimates Developed by the UN Inter-agency Group for Child Mortality Estimation (UNICEF, WHO, World Bank, UN DESA Population Division) at www.childmortality.org."
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+statType: dcs:measuredValue
+measuredProperty: dcs:mortalityRate
+measurementDenominator: dcs:Count_BirthEvent_LiveBirth
+age: dcs:YearsUpto4
+
+
+Node: dcid:WorldBank/SH_PRV_SMOK
```

Comment:
Same comment as above for this one. The existing SV is this: ``dcid:Count_Person_15OrMoreYears_Smoking_AsFractionOf_Count_Person_15OrMoreYears``

You can find this with code search in the file ``third_party/datacommons/schema/stat_vars/manual_wdi_stat_vars.mcf``